app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,28 @@
+class EntitiesController < ApplicationController
+  def new
+    @entity = Entity.new
+    @group = Group.find(params[:group_id])
+    @user = @group.author
+  end
+
+  def create
+    @entity = Entity.new(entity_params)
+    @entity.author = current_user
+
+    if @entity.save
+      # If save is successful, associate groups with the entity
+      @entity.groups << Group.find(params[:entity][:group_ids]) if params[:entity][:group_ids].present?
+
+      flash[:success] = 'Entity created successfully'
+      redirect_to entities_path
```

Comment:
- [ ] There is a redirection error when trying to create a transaction, just like in the video. Also the transaction is not created so further assessments cannot be made on that. Kindly fix that.

  ![image](https://github.com/78fatemanazari/ezPay-rails-app/assets/54848363/acfa319a-69e9-497d-bd40-b514f313969d)

Reply:
@DammyShittu 
I need a little help with this error, everything is ok, I have a join table and my routes are configured correctly. 
I reached out for help but there was no one to help me. I was stuck with this error for several days. 
Please kindly help me if you have time! Thanks